lib/bill_calculator.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'menu'
+require 'BigDecimal'
+
+class BillCalculator
+
+  attr_reader :cost
+
+  def initialize
+    @cost = 0
+    @menu = Menu.new
+  end
+
+  def bill(order)
+    calculate(order)
+    puts "Total order cost is £#{@cost.to_f}"
+  end
+
+  def calculate(order)
+    order.each do |item|
+      @cost += BigDecimal("#{@menu.dishes[item]}")
+    end
+  end
```

Comment:
Good srp here. 
I liked your approach to returning a float, I wish I had done it the same way. Much more readable